doc/examples/reconst_dki.py

```diff
@@ -251,10 +289,11 @@
 .. [Fierem2011] Fieremans E, Jensen JH, Helpern JA (2011). White matter
                 characterization with diffusion kurtosis imaging. NeuroImage
                 58: 177-188
-.. [NetoHe2014] Neto Henriques R, Ferreira HA, Correia MM (2012). Diffusion
-                kurtosis imaging of the healthy human brain. Master
-                Dissertation Bachelor and Master Program in Biomedical
-                Engineering and Biophysics, Faculty of Sciences.
+.. [NetoHe2012] Neto Henriques R, Ferreira H, Correia M, (2012). Diffusion
+                kurtosis imaging of the healthy human brain.Master Dissertation
```

Comment:
Spaces in this reference are a bit weird:

"...human brain.Master..." => "...human brain. Master..."
"...Bachelor andMaster Programin..." => "Bachelor and Master Program in..."